src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+import { getMovie } from '../../api';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  onAddMovie: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ onAddMovie }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState<string>('');
+  const [hasError, setHasError] = useState<boolean>(false);
+  const [isLoading, setLoading] = useState<boolean>(false);
```

Comment:
no need to indicate generics for primitives, those are set automatically
```suggestion
  const [query, setQuery] = useState('');
  const [hasError, setHasError] = useState(false);
  const [isLoading, setLoading] = useState(false);
```